### PR TITLE
add support aten::__ior__

### PR DESCRIPTION
**Pull request by eaidova**
### Details:
 - *add support `aten::__ior__`, `aten::__iand__`, `aten::__ixor__`*

### Tickets:
 - *CVS-156301*

### CODE REVIEW

File: src/frontends/pytorch/src/op_table.cpp [hunk 1]
```diff
@@ -331,12 +331,15 @@ OP_CONVERTER(translate_zeros_like_fx);
...
         {"aten::__and__", op::translate_bitwise_and},
+        {"aten::__iand__", op::inplace_op<op::translate_bitwise_and>},
         {"aten::__derive_index", op::translate_derive_index},
         {"aten::__getitem__", op::translate_getitem},
         {"aten::__not__", op::translate_1to1_match_1_inputs<opset10::LogicalNot>},
         {"aten::__or__", op::translate_bitwise_or},
+        {"aten::__ior__", op::inplace_op<op::translate_bitwise_or>},
         {"aten::__range_length", op::translate_range_length},
         {"aten::__xor__", op::translate_bitwise_xor},
+        {"aten::__ixor__", op::inplace_op<op::translate_bitwise_xor>},
```
Comment (rkazants) [inline src/frontends/pytorch/src/op_table.cpp:342]: can we have tests for them?

Comment (eaidova) [reply src/frontends/pytorch/src/op_table.cpp:342]: added

File: tests/layer_tests/pytorch_tests/test_bitwise_ops.py [hunk 2]
```diff
@@ -140,3 +140,61 @@ def test_bitwise_operators(self, lhs_dtype, rhs_dtype, lhs_shape, rhs_shape, ie_
...
+                return lhs.__iand__(rhs)
+    
+            def forward_xor(self, lhs, rhs):
+                return lhs.__ixor__(rhs)
+
+        return aten_bitwise(op), None, op
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize("dtype", ["bool", "int32"])
```
Comment (rkazants) [inline tests/layer_tests/pytorch_tests/test_bitwise_ops.py:176]: can we run these tests for export and fx?

**Review by rkazants — APPROVED** [review]

File: tests/layer_tests/pytorch_tests/test_bitwise_ops.py (hunk 2 above)
Comment (eaidova) [reply tests/layer_tests/pytorch_tests/test_bitwise_ops.py:176]: it does not support inplace ops as i know

Comment (rkazants) [reply tests/layer_tests/pytorch_tests/test_bitwise_ops.py:176]: right, not needed then

**Review by mvafin — APPROVED** [review]

**Pull request merged**